megalodon/src/firefish.ts

```diff
@@ -953,12 +953,26 @@ export default class Firefish implements MegalodonInterface {
   // accounts/preferences
   // ======================================
   public async getPreferences(): Promise<Response<Entity.Preferences>> {
-    return new Promise((_, reject) => {
-      const err = new NoImplementedError('firefish does not support')
-      reject(err)
+    return this.client.post<FirefishAPI.Entity.UserDetailMe>('/api/i').then(async res => {
+      return Object.assign(res, {
+        data: FirefishAPI.Converter.userPreferences(res.data, await this.getDefaultPostPrivacy())
+      })
     })
   }
 
+  public async getDefaultPostPrivacy(): Promise<'public' | 'unlisted' | 'private' | 'direct'> {
```

Comment:
This method is not defined in megalodon interface. If this method is used in only `getPereferences`, please change `public` to `private`.

Reply:
Since the function is used directly in Firefish, I defined it in the Megalodon interface.